### PR TITLE
Get the Ruby version from Java

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

**Comment by bjfish** [comment]
Trying my local MRI I've found the ruby_version in rbconfig to be 2.3.0 for even my 2.3.1 version ruby. Not sure what this would imply.

**Comment by bjfish** [comment]
To follow up on my `2.3.0` comment. I think it originates from this file: https://github.com/ruby/ruby/blob/ruby_2_3/include/ruby/version.h#L35  what they describe as a "Ruby API version"

File: truffleruby/src/main/java/org/truffleruby/language/TruffleBootNodes.java [hunk 1]
```diff
@@ -40,6 +40,17 @@
...
+    @CoreMethod(names = "ruby_version", onSingleton = true)
+    public abstract static class RubyVersionNode extends CoreMethodNode {
+
+        @TruffleBoundary
+        @Specialization
+        public DynamicObject rubyVersion() {
+            return createString(StringOperations.encodeRope(RubyLanguage.RUBY_VERSION, UTF8Encoding.INSTANCE));
+        }
+
+    }
```
Comment (eregon) [inline truffleruby/src/main/java/org/truffleruby/language/TruffleBootNodes.java]: This already exists in constant `RUBY_VERSION` :wink:

**Comment by eregon** [comment]
Fixed by 	9ef92fa.

**Pull request closed without merging**